### PR TITLE
Modoaipmh 514 - purge error logs

### PR DESCRIPTION
**Pull request by firdavs-m**
## Pre-Merge Checklist:
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] Check logging
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/main/java/org/folio/rest/impl/CleanUpErrorLogs.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+import static org.folio.oaipmh.Constants.OKAPI_TENANT;
+import static org.folio.oaipmh.Constants.REPOSITORY_FETCHING_CLEAN_ERRORS_INTERVAL;
+import static org.folio.rest.jaxrs.resource.OaiPmhCleanUpInstances.PostOaiPmhCleanUpInstancesResponse.respond500WithTextPlain;
```
Comment (siarhei-charniak) [inline src/main/java/org/folio/rest/impl/CleanUpErrorLogs.java:29]: Please move static imports to the top of the file.

**Review by siarhei-charniak — APPROVED** [review]

File: src/main/java/org/folio/rest/impl/CleanUpErrorLogs.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        } catch (Exception ex) {
+          logger.error("cannot retrieve config from system properties");
+        }
+      } else {
+        logger.error("loading configuration data failed");
+      }
+    });
+
+    OffsetDateTime offsetDateTime = ZonedDateTime
+      .ofInstant(Instant.now(), ZoneId.systemDefault())
```
Comment (obozhko-folio) [inline src/main/java/org/folio/rest/impl/CleanUpErrorLogs.java:69]: Wouldn't be better to use UTC instead of system default here?

Comment (firdavs-m) [reply src/main/java/org/folio/rest/impl/CleanUpErrorLogs.java]: fixed

File: src/main/java/org/folio/rest/impl/CleanUpErrorLogs.java (hunk 1 above)
Comment (firdavs-m) [reply src/main/java/org/folio/rest/impl/CleanUpErrorLogs.java]: fixed

**Review by obozhko-folio — APPROVED** [review]

**Pull request merged**